### ISSUE TITLE
Missing support for closing dates

### ISSUE THREAD

**Issue by curateideas** [hougaard/BCCL_support#23]
**Describe the bug**
We exported data with closing date information from SQL to a CSV-file (NAV 2018 G/L Entry Table as source). On importing these dates the closing date information is not transferred to the target date fields (with parameter ClosingDate = Yes).

**To Reproduce**
1. Command Line

`.\BCCL -t putdata -s table=$tableID -i $importfile.FullName -f csv -m $mappingfilename -l ";" -d true`

3. Mapping File

    {
      "FieldNo": 4,
      "FieldName": "Posting Date",
      "MappedName": "Posting Date",
      "Format": "dd.MM.yyyy HH:mm:ss.fff",
      "Validate": false
    },

4. Data example

"Entry No_";"G_L Account No_";"Posting Date";"Document Type";"Document No_";"Description";"Bal_ Account No_";"Amount";"Global Dimension 1 Code";"Global Dimension 2 Code";"Quantity";"VAT Amount";"Gen_ Posting Type";"Gen_ Bus_ Posting Group";"Gen_ Prod_ Posting Group";"Bal_ Account Type";"Debitor Amount";"Credit Amount";"Document Date";"External Document No_";"Source Type";"Source No_";"VAT Bus_ Posting Group";"VAT Prod_ Posting Group";"Reversed"
"1";"0090";"31.12.2018 23:59:59.999";"0";"START";"Eröffnungsposten";"";"2052148.41000000000000000000";"";"";"0.00000000000000000000";"0.00000000000000000000";"0";"";"";"0";"2052148.41000000000000000000";"0.00000000000000000000";"31.12.2018";"";"0";"";"";"";"0"

5. See error

No error message during import but missing "C" for closing date information in the imported record:

![image](https://github.com/hougaard/BCCL_support/assets/58227881/8c22d34c-b4b4-4aa2-8525-1877f319db47)

**Expected behavior**
Date values should be presented as closing dates.
**Desktop (please complete the following information):**
 - OS: Windows
 - Version 180

**Comment by hougaard** [comment]
I'm out today, I'll check tomorrow....

**Comment by hougaard** [comment]
Yeah, that seems to be stripped in the date conversion toolchain.

I have made a fix, and it will trickle to appsource "soon".

**Comment by curateideas** [comment]
Did the link to the download section change? 

When I enter https://www.hougaard.com/download/

I get this site:

![image](https://github.com/hougaard/BCCL_support/assets/58227881/ff7a2ac4-4c16-4b1d-b633-3dda44039f2a)

**Comment by curateideas** [comment]
Can you give me a hint for formatting date values when I directly import from SQL? If I omit the "Format" in the mapping, I get error: An error has occurred: Unable to convert from Microsoft.Dynamics.Nav.Runtime.NavJsonValue to Microsoft.Dynamics.Nav.Runtime.NavDate.

**Comment by hougaard** [comment]
What field types are you trying to connect? (On both sides)?

**Comment by curateideas** [comment]
SQL: Date (Standard field "Posting Date" in Table 17) or is it DateTime since the SQL field type determines the format?

Target BC: Date ("Posting Date" in a new Table)
